examples/hubert/utils/feature_utils.py

```diff
@@ -99,7 +141,11 @@ def dump_features(
             Options: [``torch.device('cpu')``, torch.device('cuda')``].
         feature_type (str, optional): The type of the desired feature. Options: [``mfcc``, ``hubert``].
             (Default: ``mfcc``)
-        sample_rate (int, optional): The sample rate of the audio. (Default: 16000)
+        layer_index (int, optional): The index of transformer layers in ``torchaudio.models.HuBERTPretrainModel``
```

Comment:
would it make more sense for this to be an `Optional[int]` type if it's unused if `feature_type` is not `hubert`?

Reply:
Good point, will update it.